ParisBNF/et/BNFet4.xml

```diff
@@ -111,6 +113,15 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                   ነጋድያነ፡ ኢየሩሳሌም፡ ዝክሩኒ፡ በጸሎትክሙ፡ ኢትርሰኡኒ፡ ወትረ፡ ለእመ፡ ነጸርክሙ፡
                   መ<supplied reason="undefined" resp="PRS10747Zotenbe">ጽሐ</supplied>ፈ፡ ኦሪተ።</q>
                         </item>
+                        <item xml:id="e1">
+                           <desc>The manuscript has  short verses of the Bible in Hebrew but written in Ethiopic characters in the upper margins on ff. <locus target="#22v">22v</locus>, <locus target="#90v">90v</locus></desc>
```

Comment:
maybe this is an addition rather than varia?
And unless someone is already planning to come back to the manuscript after the article is published, I think it would be good know to have the source of this information outside of a comment, even if there is no zotero tag yet, just having it in a note would be helpful for users.

Reply:
there was a comment to come back later to but ok to a note